Overhaul aws index (#344)
AlexsLemonade/OpenScPCA-analysis: docs/software-platforms/aws/index.md

We use Amazon Web Services (AWS) to grant contributors access to data and Linux virtual computers.

!!! note
    See [Getting Access to Resources](../../getting-started/accessing-resources/index.md) for more information about getting AWS access.

>     Once we have created an AWS account for you, you can proceed to [set up your account](./joining-aws.md).

Reviewer: Since I was assigned to #258, I started to stack on this... but I no longer think that makes sense. Can you add (suggestion is not working for me) the following here:

```
    Your use of AWS services described in this section counts towards your [monthly budget](../../getting-started/accessing-resources/getting-access-to-compute.md#monthly-budget).
```

Author: > but I no longer think that makes sense. Can you add (suggestion is not working for me) the following here:

Replace or just add?

Reviewer: Add

## S3: AWS for data

We use [AWS S3](https://aws.amazon.com/s3/) to store project data in folders, referred to in S3 as "buckets."
Please refer to [our documentation on accessing project data](../../getting-started/accessing-resources/getting-access-to-data.md#accessing-data-on-s3) for more information.

We also use S3 as central location for you to share your result files with us before [filing a pull request](../../contributing-to-analyses/creating-pull-requests/index.md).
All contributors will have a designated [researcher bucket](working-with-s3-buckets.md) where results can be synced and shared as part of [code review](../../contributing-to-analyses/pr-review-and-merge/index.md).

## LSfR: AWS for virtual computing

> As an OpenScPCA contributor, you are eligible for a monthly allocation on [Lightsail for Research (LSfR)](https://aws.amazon.com/lightsail/research/), a Amazon product that allows you to work on a virtual Ubuntu (Linux) Desktop computer in your browser.

Reviewer: ```suggestion
As an OpenScPCA contributor, you are eligible for a monthly allocation on [Lightsail for Research (LSfR)](https://aws.amazon.com/lightsail/research/), an Amazon product that allows you to work on a virtual Ubuntu (Linux) Desktop computer in your browser.
```

We use Amazon Web Services (AWS) to grant contributors access to data and Linux virtual computers.

!!! note
    See [Getting Access to Resources](../../getting-started/accessing-resources/index.md) for more information about getting AWS access.

    Once we have created an AWS account for you, you can proceed to [set up your account](./joining-aws.md).
    Note that your use of AWS services described in this section counts towards your [monthly budget](../../getting-started/accessing-resources/getting-access-to-compute.md#monthly-budget).

> 

Reviewer 2: ```suggestion
## S3: Storage with AWS
```
Something about these headers don't make complete sense to me.

## S3: AWS for data

We use [AWS S3](https://aws.amazon.com/s3/) to store project data in folders referred to in S3 as "buckets."
Please refer to [our documentation on accessing project data](../../getting-started/accessing-resources/getting-access-to-data.md#accessing-data-on-s3) for more information.

We also use S3 as central location for you to share your result files with us before [filing a pull request](../../contributing-to-analyses/creating-pull-requests/index.md).
All contributors will have a designated [researcher bucket](working-with-s3-buckets.md) where results can be synced and shared as part of [code review](../../contributing-to-analyses/pr-review-and-merge/index.md).

> 

Reviewer 2: ```suggestion
## LSfR: Virtual computing with AWS
```

## LSfR: AWS for virtual computing

> 

Reviewer 2: I think I would add a sentence here about why you would want to use lightsail (access to computing resources so you don't have to use your own). And then a bulleted list of what you will need to set up with links to the docs. 

I would probably do the same with a bulleted list for S3 with links to each of the docs?

Author: > a bulleted list for S3 with links to each of the docs

For this, I'll link both the one doc in this section, and relevant docs pages from other sections.

We use Amazon Web Services (AWS) to grant contributors access to data and Linux virtual computers.

!!! note
    See [Getting Access to Resources](../../getting-started/accessing-resources/index.md) for more information about getting AWS access.

    Once we have created an AWS account for you, you can proceed to [set up your account](./joining-aws.md).

## S3: AWS for data

> We use [AWS S3](https://aws.amazon.com/s3/) to store project data in files and folders known in S3 lingo as "buckets."

Reviewer 2: ```suggestion
We use [AWS S3](https://aws.amazon.com/s3/) to store project data in folders, referred to in S3 as "buckets."
```

Please refer to [our documentation on accessing project data](../../getting-started/accessing-resources/getting-access-to-data.md#accessing-data-on-s3) for more information.

We also use S3 as central location for you to share your result files with us before [filing a pull request](../../contributing-to-analyses/creating-pull-requests/index.md).
All contributors will have a designated [researcher bucket](working-with-s3-buckets.md) where results can be synced and shared as part of [code review](../../contributing-to-analyses/pr-review-and-merge/index.md).

## LSfR: AWS for virtual computing

> We use an AWS product called ["Lightsail for Research" (LSfR)](https://aws.amazon.com/lightsail/research/) to provide virtual Linux (specifically, Ubuntu) computers.

Reviewer 2: I would take what's currently in `software-platforms/index.md` and replace this with that. Then you won't need to come back in and fill this in.

Author: Might as well take this change to quickly clean up `software-platforms/index.md` too!

## S3: Data storage with AWS

We use [AWS S3](https://aws.amazon.com/s3/) to store project data in folders referred to in S3 as "buckets."
We also use S3 as central location for you to share your result files with us before [filing a pull request](../../contributing-to-analyses/creating-pull-requests/index.md).
All contributors will have a designated [researcher bucket](working-with-s3-buckets.md) where results can be synced and shared as part of [code review](../../contributing-to-analyses/pr-review-and-merge/index.md).

Please refer to these pages about working with S3:

- [Locally logging into AWS](../../technical-setup/environment-setup/configure-aws-cli.md#logging-in-to-a-new-session)
- [Accessing project data from S3](../../getting-started/accessing-resources/getting-access-to-data.md#accessing-data-on-s3)
- [Using your researcher bucket](working-with-s3-buckets.md)

## LSfR: Virtual computing with AWS

As an OpenScPCA contributor, you are eligible for a monthly allocation on [Lightsail for Research (LSfR)](https://aws.amazon.com/lightsail/research/), an Amazon product that allows you to work on a virtual Ubuntu (Linux) Desktop computer in your browser.

> Please refer to these pages about using LSfR:

Reviewer 2: ```suggestion
Please refer to these pages to learn more about setting up LSfR:
```
Since they need to follow these links to get set up (or at least the first three) what do we think about rephrasing this?

Author: how about `...to learn more about setting up and using LSfR`?

Author: https://github.com/AlexsLemonade/OpenScPCA-analysis/pull/344/commits/c9a0231c7785dc4f415f2ebfe52febaf8921f025

Reviewer 2: I like this!
